src/fal/packages/environments/virtual_env.py

```diff
@@ -2,34 +2,35 @@
 
 import hashlib
 import os
+import shutil
 import subprocess
 import sysconfig
-import shutil
 from dataclasses import dataclass, field
 from pathlib import Path
 from typing import ContextManager, List
 
 from fal.packages import bridge, isolated_runner
-from fal.packages.dependency_analysis import get_default_requirements
+from fal.packages.dependency_analysis import get_default_pip_dependencies
 from fal.packages.environments.base import (
     BASE_CACHE_DIR,
     BaseEnvironment,
     IsolatedProcessConnection,
     log_env,
     rmdir_on_fail,
 )
+from fal.utils import cache_static
 
 _BASE_VENV_DIR = BASE_CACHE_DIR / "venvs"
 _BASE_VENV_DIR.mkdir(exist_ok=True)
 
 
 @dataclass
 class VirtualPythonEnvironment(BaseEnvironment[Path], make_thread_safe=True):
-    requirements: List[str] = field(default_factory=list)
+    pip_requirements: List[str] = field(default_factory=list)
```

Comment:
ATL: would we change this?

Reply:
Fixed now, so venv uses `requirements` and conda uses `packages` with both in config and in the internal implementation.